PWGCF/FemtoWorld/Tasks/femtoWorldPairTaskTrackD0.cxx

```diff
@@ -300,15 +296,15 @@ struct femtoWorldPairTaskTrackD0 {
 
     for (auto& d0d0bar : groupPartsD0D0bar) {
       if (d0d0bar.flagD0() == 1) {
-        registry.fill(HIST("hInvMassD0"), d0d0bar.mass());
+        registry.fill(HIST("hInvMassD0"), d0d0bar.massD0());
         registry.fill(HIST("hMomentumD0"), d0d0bar.p());
         registry.fill(HIST("hPtD0"), d0d0bar.pt());
         registry.fill(HIST("hPhiD0"), d0d0bar.phi());
         registry.fill(HIST("hEtaD0"), d0d0bar.eta());
         registry.fill(HIST("hDecayLengthD0"), d0d0bar.decayLength());
       }
       if (d0d0bar.flagD0bar() == 1) {
-        registry.fill(HIST("hInvMassD0bar"), d0d0bar.mass());
+        registry.fill(HIST("hInvMassD0bar"), d0d0bar.massD0bar());
```

Comment:
In the future please minimize the number of rows ;) maybe put a mass of D0 to the mass() and create a d0barmass() for the D0bar. It will be one row less in that way :)